### PR TITLE
implement pkg handler exists test cases

### PR DESCRIPTION
**Pull request by kmrmt**
Signed-off-by: Kosuke Morimoto <ksk@vdaas.org>

### Description:

SSIA

### Environment:

- Go Version: 1.18.1
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [X] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vankichi** [comment]
It's fine about the test case.

File: pkg/agent/core/ngt/handler/grpc/handler_test.go [hunk 1]
```diff
@@ -365,6 +365,32 @@ func Test_server_Exists(t *testing.T) {
 		}
 		return nil
 	}
+	/*
+		Exists test cases (focus on ID(string), only test float32):
+		- Equivalence Class Testing ( 1000 vectors inserted before a search )
+			- case 1.1: success exists vector
+			- case 2.1: fail exists with non-existent ID
+		- Boundary Value Testing ( 1000 vectors inserted before a search )
+			- case 1.1: fail search with ""
```
Comment (kevindiu) [inline pkg/agent/core/ngt/handler/grpc/handler_test.go:374]:
```suggestion
			- case 1.1: fail exists with ""
```

**Review by kevindiu — DISMISSED** [review]
LGTM

**Review by vankichi — COMMENTED** [review]
Could you please fix some missed spells?
Almost LGTM!

**Review by vankichi — APPROVED** [review]

**Review by kevindiu — APPROVED** [review]
LGTM
